src/ui/components/DropdownMenuItem/index.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+
+import Link from 'amo/components/Link';
+
+import './styles.scss';
+
+
+type Props = {|
+  children: string | Link,
```

Comment:
This looks like it should be `children?:` since it's optional for the case when `onClick` is defined, I think?